llpc/context/llpcCompiler.cpp

```diff
@@ -172,6 +172,9 @@ opt<bool> EnablePerStageCache("enable-per-stage-cache", cl::desc("Enable shader
 opt<int> ContextReuseLimit("context-reuse-limit",
                            cl::desc("The maximum number of times a compiler context can be reused"), init(100));
 
+// -fatal-llvm-errors: Make all LLVM errors fatal
+opt<bool> FatalLLVMErrors("fatal-llvm-errors", cl::desc("Make all LLVM errors fatal"), init(false));
```

Comment:
FatalLLVMErrors -> FatalLlvmErrors

We use camelcase.

Reply:
Changed to camelcase.

Reply:
Thanks